Tests/scripts/hook_validations/integration.py

```diff
@@ -66,9 +66,46 @@ def is_valid_integration(self):
         """Check whether the Integration is valid or not, update the _is_valid field to determine that"""
         self.is_valid_subtype()
         self.is_default_arguments()
+        self.is_proxy_configured_correctly()
+        self.is_insecure_configured_correctly()
 
         return self._is_valid
 
+    def is_valid_param(self, param_name, param_display):
+        """Check if the given parameter has the right configuration."""
+        err_msgs = []
+        configuration = self.current_integration.get('configuration', [])
+        for configuration_param in configuration:
+            param_name = configuration_param['name']
+            if param_name == param_name:
+                if configuration_param['display'] != param_display:
+                    err_msgs.append('The display name of the {} parameter should be \'{}\''.format(param_name,
+                                                                                                   param_display))
+
+                elif configuration_param.get('defaultvalue', '') != '':
+                    err_msgs.append('The default value of the {} parameter should be \'\''.format(param_name))
+
+                elif configuration_param.get('required', False):
+                    err_msgs.append('The required field of the {} parameter should be False'.format(param_name))
+
+                elif configuration_param.get('type') != 8:
+                    err_msgs.append('The type field of the {} parameter should be 8'.format(param_name))
+
+        if err_msgs:
+            print_error('Received the following error for {} validation:\n'.format(param_name, '\n'.join(err_msgs)))
+            self._is_valid = False
+            return False
+
+        return True
+
+    def is_proxy_configured_correctly(self):
+        """Check that if an integration has a proxy parameter that it is configured properly."""
+        return self.is_valid_param('proxy', 'Use system proxy settings')
+
+    def is_insecure_configured_correctly(self):
+        """Check that if an integration has a insecure parameter that it is configured properly."""
+        return self.is_valid_param('insecure', 'Trust any certificate (unsecure)')
```

Comment:
in some integration the parameter is called `unsecure`, let add this option as well